go/teams/member_test.go

```diff
@@ -152,6 +152,41 @@ func TestMemberChangeRole(t *testing.T) {
 	assertRole(tc, name, other.Username, keybase1.TeamRole_READER)
 }
 
+func TestMemberRemoveRotatesKeys(t *testing.T) {
+	tc, owner, other, name := memberSetupMultiple(t)
+	defer tc.Cleanup()
+
+	before, err := Get(context.TODO(), tc.G, name)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if before.Box.Generation != 1 {
+		t.Fatalf("initial team generation: %d, expected 1", before.Box.Generation)
+	}
+
+	if err := SetRoleWriter(context.TODO(), tc.G, name, other.Username); err != nil {
+		t.Fatal(err)
+	}
+	if err := RemoveMember(context.TODO(), tc.G, name, other.Username); err != nil {
+		t.Fatal(err)
+	}
+
+	assertRole(tc, name, owner.Username, keybase1.TeamRole_OWNER)
+	assertRole(tc, name, other.Username, keybase1.TeamRole_NONE)
+
+	after, err := Get(context.TODO(), tc.G, name)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if after.Box.Generation != 2 {
+		t.Errorf("after member remove: team generation: %d, expected 2", after.Box.Generation)
+	}
+
+	if after.Box.Ctext == before.Box.Ctext {
+		t.Error("TeamBox.Ctext did not change when member removed")
+	}
```

Comment:
Ok, let's cover the end-to-end that the box for the dropped user was dropped in [this internal ticket|https://keybase.atlassian.net/browse/CORE-5032]